### PR TITLE
Upgrade Prettier to Prettier 1.9

### LINKED ISSUE #3385: Deprecation warning is printed in master
```
~/forks/prettier master ⇡ 
❯ git log -1
commit ceabc48e01b4b8977ae98f0b4bc774bd2659a294 (HEAD -> master, upstream/master, gql-cooked)
Merge: c1623d88 24036480
Author: Lucas Duailibe <lucasds@gmail.com>
Date:   2017-12-03 03:25:17 -0300

    Format CSS prop for emotion (#3381)

~/forks/prettier master ⇡ 
❯ echo a | ./bin/prettier.js 
[warn] `proseWrap` with value `false` is deprecated. Prettier now treats it as: `{ proseWrap: "never" }`.
a;

```

### PR DESCRIPTION
**Pull request by azz**
Fixes #3385

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,3 +0,0 @@
-{
-  "proseWrap": false
```
Comment (ikatyang) [inline .prettierrc]: `"proseWrap": "never"`?

Comment (azz) [reply .prettierrc]: I'm happy with the default?

Comment (ikatyang) [reply .prettierrc]: I'm fine with both `"preserve"` (default) and `"never"`, just want to make sure it's intended.

Comment (azz) [reply .prettierrc]: Yeah it was intentional.

**Comment by azz** [comment]
Wow - I got a merge failed error because we clicked merge at the same time. Never seen that before!

**Pull request merged**